virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY                                     
+[id="virt-hot-plugging-network-interfaces"]                            
+= Hot plugging secondary network interfaces                                            
+include::_attributes/common-attributes.adoc[]                  
+:context: virt-hot-plugging-network-interfaces                        
+                                                                
+toc::[]     
+
+You can add or remove secondary network interfaces without stopping your virtual machine (VM). {VirtProductName} supports hot plugging and hot unplugging for Linux bridge interfaces that use the VirtIO device driver.
+
+:FeatureName: Hot plugging and hot unplugging bridge network interfaces
+include::snippets/technology-preview.adoc[]
+
+[id="virtio-limitations_{context}"]
+== VirtIO limitations
+Each VirtIO interface uses one of the limited Peripheral Connect Interface (PCI) slots in the VM. There are a total of 32 slots available. The PCI slots are also used by other devices and must be reserved in advance, therefore slots might not be available on demand. {VirtProductName} reserves up to four slots for hot plugging interfaces. This includes any existing plugged network interfaces. For example, if your VM has two existing plugged interfaces, you can hot plug two more network interfaces. 
+
+[NOTE]
+====
+The actual number of slots available for hot plugging also depends on the machine type. For example, the default PCI topology for the q35 machine type supports hot plugging one additional PCIe device. For more information on PCI topology and hot plug support, see the link:https://libvirt.org/pci-hotplug.html[libvirt documentation].
+====
+
+If you restart the VM after hot plugging an interface, that interface becomes part of the standard network interfaces. 
```

Comment:
I'm afraid that even after rewording this sentence, it still implies that in order for the hot-plugged interface to become part of the standard network interfaces - you **must** restart the VM.
Perhaps replace the word `becomes` with `remains`?

Reply:
But it doesn't remain, it becomes.
For example q35 adds one extra slot. On hotplug this extra slot will be used, but after reboot since the hotplugged interface becomes part of the standard network interfaces, an extra slot will be added.

Reply:
I guess that when 2 people tell you your are drunk - then you should go to sleep (sorry, it sounds better in Hebrew...).
So it's probably me that didn't understand this sentence correctly. Please ignore my comment then, and sorry for the noise.
And thanks for the clarification, Alona.